services/web/src/modals/EditUser.js

```diff
@@ -64,18 +63,15 @@ export default class EditUser extends React.Component {
   };
 
   render() {
-    const { user, touched, loading, error } = this.state;
+    const { user, loading, error } = this.state;
     return (
       <>
         <Modal.Header>
-          {this.isUpdate() ? `Edit "${user.name}"` : 'New User'}
+          {this.isUpdate() ? `Edit "${user.name || user.email}"` : 'New User'}
```

Comment:
you made the name fields required now so they should always have a name defined I think?

Reply:
i made name not required, to deal with invites ...

Reply:
ok gotcha, must have misread that